### DISCUSSION TITLE
The title of a plot shows runtime of the cell. Can I see the date of the satellite image on the title instead of runtime?

### DISCUSSION THREAD

**Discussion by yujulitengigant** [DigitalEarthSweden/digital-earth-sweden-community#1] [Q&A]
Hi!

I have checked DES OpenEO demo scripts. 
When I put a spatial and temporal input data to load a satellite image (s2 or s3) with matplotlib, the title of the plot shows runtime of the cell. Instead, can I see the date of the satellite image on the plot so that I can check which date the image is on? Thank you!

<img width="544" alt="bild" src="https://user-images.githubusercontent.com/97133478/208096485-a1bba136-9926-4f46-a024-23ee59c90552.png">

**Comment by danielFlemstrom** [comment]
Unfortunately, the underlying packages changed this behavior. We will investigate this matter after the holidays!

**Reply by Yujuson** [reply]
Thank you! I'll check it again after holidays.
